cli/internal/helm/client.go

```diff
@@ -240,43 +249,31 @@ func (c *Client) currentVersion(release string) (string, error) {
 	return rel[0].Chart.Metadata.Version, nil
 }
 
-// ServiceVersions bundles the versions of all services that are part of Constellation.
-type ServiceVersions struct {
-	cilium                 string
-	certManager            string
-	constellationOperators string
-	constellationServices  string
-	awsLBController        string
-}
-
-// NewServiceVersions returns a new ServiceVersions struct.
-func NewServiceVersions(cilium, certManager, constellationOperators, constellationServices string) ServiceVersions {
-	return ServiceVersions{
-		cilium:                 cilium,
-		certManager:            certManager,
-		constellationOperators: constellationOperators,
-		constellationServices:  constellationServices,
+func (c *Client) csiVersions() (map[string]string, error) {
+	packedChartRelease, err := c.actions.listAction(csiInfo.releaseName)
+	if err != nil {
+		return nil, fmt.Errorf("listing %s: %w", csiInfo.releaseName, err)
 	}
-}
 
-// Cilium returns the version of the Cilium release.
-func (s ServiceVersions) Cilium() string {
-	return s.cilium
-}
+	// No CSI driver installed
+	if len(packedChartRelease) == 0 {
+		return nil, nil
```

Comment:
`return nil, nil` always looks weird. Should we maybe return a "ChartNotInstalledErr" that we gracefully handle in the caller?

Reply:
Returning an error here makes no sense, since we simply use the output to fill a struct.
The "returned value" is handled [here](https://github.com/edgelesssys/constellation/blob/b0ef84e69abfbf95e837cf4fd476a04d070be7ab/cli/internal/helm/serviceversion.go#L37).

Reply:
While I think it might be ok-ish in this situation, it also opens up the possibility of people orienting themselves by this code. I had a look at a more general refactoring along the lines of making this function private and moving it to / calling it from `.String()` and `ConstellationServices()`. We don't seem to any other operation on the returned values than calling those functions (also only in 1 place each).